ide/db/src/org/netbeans/modules/db/explorer/node/ProcedureNode.java

```diff
@@ -657,29 +672,26 @@ public boolean isEditSourceSupported() {
         @Override
         public String getDDL() {
             StringBuilder expression = new StringBuilder();
-            // set delimiter
             expression.append("DELIMITER ").append(DELIMITER).append(NEW_LINE); // NOI18N
-            // DDL
-            expression.append("CREATE OR REPLACE ").append(getSource());
+            expression.append("CREATE OR REPLACE ").append(getSource()); // NOI18N
             expression.append(SPACE).append(DELIMITER).append(NEW_LINE); // NOI18N
-            // unset delimiter
             expression.append("DELIMITER ; ").append(NEW_LINE); // NOI18N
             return expression.toString();
         }
 
         private String fqn(String source, String owner) {
             String upperSource = source.toUpperCase();
-            String toFind = getTypeName(getType()) + " "; // NOI18N
+            String toFind = getTypeName(getType()) + " ";
```

Comment:
//NOI18N was dropped

Reply:
No. Strings might be recognized by tools even if it is blank. Please don't remove it. Basically, please leave it as it is.